airbyte-cdk/python/airbyte_cdk/logger.py

```diff
@@ -62,10 +60,10 @@ def format(self, record: logging.LogRecord) -> str:
             message = super().format(record)
             message = filter_secrets(message)
             log_message = AirbyteMessage(type="LOG", log=AirbyteLogMessage(level=airbyte_level, message=message))
-            return log_message.json(exclude_unset=True)
+            return log_message.json(exclude_unset=True)  # type: ignore
```

Comment:
a better end state would be to convert this to a string, maybe using `json.dumps`. I decided to leave this out of scope because to because the blast radius is potentially large.

Reply:
It looks like `.json` ends in a call to `json.dumps`. What type are you ending up with that fails mypy? 

Leaving it out of scope makes sense, I'm fine with that for now 👍🏻 Just wondering how we ended up getting that type. If you want to fix it, you could do it in a new branch below to keep the diffs separate (I use stacking for this use case a lot)

Reply:
the output value is showing up as `Any`. It might just be because it's missing type annotations?

Reply:
Huh, perhaps! Don't get too stuck on it, we can look into it later